hack/util.sh

```diff
@@ -641,7 +641,11 @@ function install_registry {
 }
 
 function wait_for_registry {
-	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template="{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}" --config=${ADMIN_KUBECONFIG} || echo "0")" != "0" ]]' $((5*TIME_MIN))
+	local config="--config=${ADMIN_KUBECONFIG}"
+	local tmplsubsets='{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}'
```

Comment:
@deads2k thinking about this, does it make sense to have an `oadm registry is-up` command or something in the CLI to facilitate this? Won't customers doing custom install/setup scripts end up building their own version of this function?